app/views/passengers/_form.html.haml

```diff
@@ -1,15 +1,21 @@
-=form_with model: @passenger do |form|
-  - if @passenger.errors.any?
-    #error_explanation
-      %h2= "#{pluralize(@passenger.errors.count, "error")} prohibited this passenger from being saved:"
-      %ul
-        - @passenger.errors.full_messages.each do |message|
-          %li= message
-
+- if @passenger.errors.any?
+  #error_explanation
+    %h2= "#{pluralize(@passenger.errors.count, "error")} prohibited this passenger from being saved:"
+    %ul
+      - @passenger.errors.full_messages.each do |message|
+        %li= message
+= form_with model: @passenger, local: true do |form|
+  .field
+    = form.label :permanent_passenger
+    = form.check_box :permanent, disabled: @current_user.dispatcher?,
+      id: "passenger_permanent"
   .field
     = form.label :passenger_name
```

Comment:
Labels and text fields should have matching arguments. If you want to change the text that's shown, use the locales file.